src/libraries/System.Private.CoreLib/src/System/Threading/Thread.cs

```diff
@@ -153,8 +153,28 @@ public Thread(ParameterizedThreadStart start, int maxStackSize)
 #if !TARGET_BROWSER
```

Comment:
I suspect you will need a dummy implementation for Mono Browser.

Reply:
Yup